meta-mender-core/recipes-mender/mender-shell/mender-shell.bb

```diff
@@ -18,6 +18,7 @@ DEPENDS_append = " glib-2.0"
 RDEPENDS_${PN} = "glib-2.0"
 
 MENDER_SERVER_URL ?= "https://docker.mender.io"
+MENDER_SHELL_USER ??= "nobody"
```

Comment:
Question:
Any reason for 'super-soft' here? As opposed to the other soft-sets?

Reply:
I think we need to be "super-soft" here to be able to be "soft" [here](https://github.com/mendersoftware/meta-mender/pull/1228/files#diff-0556501735020bdd41abfd2108411634ce0b5788f4266bd1c0e3d6df9a6d5040R1). It didn't work otherwise.